legacy/setup-opensuse.sh

```
@@ -1,15 +1,35 @@
+      gzip help2man hostname htop hwloc hwloc-devel joe m4 make libbsd-devel libtool libicu-devel \
+      glu-devel grpc-devel libunistring-devel libuuid-devel libX11-devel libXau-devel \
+      libXdmcp-devel libXext-devel libXfont-devel libXft-devel libxml2-devel \
+      libXmu-devel libXpm-devel libXrender-devel liblz4-devel libzstd-devel lsb-release Mesa-devel \
+      libXmu-devel libXpm-devel libXrender-devel liblz4-devel libzstd-devel lsb-release Mesa-devel \
+      ncurses-devel openssl-devel patch procps protobuf-devel python3 python3-devel \
+      readline-devel rsync sed subversion tar libtbb2 tbb-devel unzip wget which \
+      libxerces-c-devel xz xz-devel yaml-cpp-devel
 zypper clean
+
+os_version=${cat /etc/os-release | grep 'VERSION=' | cut -d\" -f2}
+if [[ "{{ os_version }}" == "15.6" ]]; then
```

Shellcheck (0.11.0)

[warning] This expression is constant. Did you forget the $ on a variable?

(SC2050)